content/content-strategy/audit-content/prove-value/inefficiencies.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: text/textblock
+---
+
+To prove the benefits of a content audit, show how much you can save by fixing bad content.
+
+To show costs and efficiencies it helps to highlight what it costs your agency to:
+handle calls relating to the clarification or lack of web content
+double-handle applications that are incorrectly completed as a result of out-of-date or unclear information
+police and prosecute non-compliance that can be avoided if people know what to do
+manage and maintain your current website and the outcomes you’re getting for that investment
+
+You could calculate a ‘cost per view rate’ by collating the yearly costs of managing and maintaining your site. Costs could include technical, staffing and content development. Then divide this by the number of views you get from users.
+
+Use your data on user-satisfaction rates to plot the yearly spend against user-satisfaction rates. See if there are any relationships or trends.
+
+Draw up a map of the processes you use for managing content development and publishing. This is a way of finding any inefficiencies or blockages in the current process, that you could  improve upon.
```

Comment:
Minor typo: double space between 'could' and 'improve upon.'

Reply:
Trisha fixed